package.json

```diff
@@ -128,14 +128,14 @@
     "opener": "^1.5.1",
     "prettier": "^1.16.4",
     "prop-types": "^15.6.2",
-    "react": "0.0.0-4221565e1",
+    "react": "^0.0.0-4221565e1",
```

Comment:
This is suspicious to me. Last time I did that, it gave me a completely different beta version from months ago because it matches alphabetically (its commit hash was later in the alphabet).

**You can verify that `npm install react@^0.0.0-4221565e1` installs `react@0.0.0-fec00a869` — not what you want.** (Although your PR is fine because the Yarn lockfile currently already has the right version.)

Instead, I think what you want is to lock down *both* React packages and the scheduler on the exact canary we want. Without the caret.

Reply:
Ah, right that makes sense. This is what I got by running `yarn add react@canary react-dom@canary scheduler@canary`.